WordPress/Classes/ViewRelated/Notifications/Views/ShareFooterView.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+import DesignSystem
+
+struct ShareFooterView: View {
+    private let title: String
+    private let onShareClicked: () -> Void
+    @Environment(\.colorScheme) var colorScheme
+
+    init(kind: ShareFooterView.Kind, onShareClicked: @escaping () -> Void) {
+        self.onShareClicked = onShareClicked
+        switch kind {
+        case .comment:
+            title = Strings.shareCommentTitle
+        case .post:
+            title = Strings.sharePostTitle
+        case .blog:
+            title = Strings.shareBlogTitle
+        }
+    }
+
+    var body: some View {
+        VStack {
+            if colorScheme == .light {
+                Divider()
+                    .frame(height: 1)
+                    .background(Color.DS.Background.secondary)
+            }
+            DSButton(
+                title: title,
+                iconName: .blockShare,
+                style: .init(emphasis: .primary, size: .large),
+                action: onShareClicked
+            )
+            .padding(.top, .DS.Padding.double)
+            .padding(.horizontal, .DS.Padding.medium)
+            .padding(.bottom, .DS.Padding.medium)
+        }.background(colorScheme == .light ? Color.DS.Background.primary : Color.DS.Background.secondary)
```

Comment:
[Suggestion] Why not use the background style that's used in the `CommentModerationView`?
```swift
.background(Color.DS.Background.primary)
```
The color itself changes depending on the color scheme.